app/scripts/controllers/registerworkflow.js

```diff
@@ -47,6 +47,12 @@ angular.module('dockstore.ui')
           });
       };
 
+      $scope.changeExt = function(name, extension){
+        var length = name.length;
+        var nameNoExt = name.substring(0,length-3);
```

Comment:
Try handling a few more cases here
For example:
- the user decided not to use an extension
- in the future, we may want to use other extensions that are not 3 characters (total example: example.yaml)

For example, look for a period, but make sure it doesn't fail with a short name or a name with no extension.